Gemfile

```diff
@@ -120,6 +120,10 @@ gem 'holidays'
 # csv
 gem 'csv'
 
+# write xlsx
+gem 'caxlsx'
+gem 'caxlsx_rails'
```

Comment:
**nitpick (blocking):** I don't believe this gem is necessary. From it's description it is only needed if rendering `xlsx` from the view with an `.xlsx.axlsx` extension view file.

Reply:
Gem removed in 27a668bc88ef615acaf70bd9192fb9979d843daeG

Validation moved in b621d77e9e5b8dbe6d4afaeff2294cdfe9b9323b